cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -228,6 +231,11 @@ func (sub EthereumSub) Start(completionEvent *sync.WaitGroup) {
 			// The user who starts this must provide a valid last processed block
 			if lastProcessedBlock.Cmp(big.NewInt(0)) == 0 {
 				lastProcessedBlock.Sub(endingBlock, big.NewInt(1))
+			} else if catchUpNeeded {
```

Comment:
This isn't part of the current change, but it's related.  On line 217, we do:

`startingBigInt := newHead.Number
`

Problem is that we're now going to change the number that's inside newHead (since newHead.Number is a pointer to an int, not an int) - that could potentially cause later problems.  I think we need:

`startingBigInt := big.NewInt(newHead.Number.Int64())`
			
unless I'm reading how go does pointers incorrectly.

Reply:
New head gets reset any time this channel receives information so we don't need to worry about pointers.